### PR TITLE
Migrate Data of Tables directly

### PR DESCRIPTION
**Pull request by ondrajodas**
Jira - https://keboola.atlassian.net/browse/COM-1343

- [x] Nejdříve mergnout - https://github.com/keboola/app-project-migrate-tables-data/pull/1

### CODE REVIEW

File: src/Migrate.php [hunk 1]
```diff
@@ -102,6 +112,23 @@ private function restoreDestinationProject(array $restoreCredentials): void
...
+        $this->logger->info('Migrate data of tables directly.');
+
+        $this->destJobRunner->runJob(
+            Config::DATA_OF_TABLES_MIGRATE_COMPONENT,
+            [
+                'parameters' => [
+                    'sourceKbcUrl' => $this->sourceProjectUrl,
+                    '#sourceKbcToken' => $this->sourceProjectToken,
+                ],
+            ]
```
Comment (webrouse) [inline src/Migrate.php:126]: Netreba tam niekde esten `runId`? Aby sme vedeli ako job vznikol? Niekde sa to tak robilo, nie som si isty ....

Comment (webrouse) [reply src/Migrate.php:126]: Mozno sa spytat v PS?

Comment (ondrajodas) [reply src/Migrate.php:126]: @webrouse no ten se nastavuje tady:
https://github.com/keboola/app-project-migrate/blob/master/src/JobRunner/SyrupJobRunner.php#L35

Comment (webrouse) [reply src/Migrate.php:126]: jj, ale nemal by sa pouzit rovnaky? tj. aby tie jobs, mali rovnake `RUN_ID`?
Asi nic, ... mal to tak asi iba stary gooddata wrier. Ok. :+1:

**Pull request merged**